pkgs/by-name/ra/radicle-node/update.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p coreutils gnused common-updater-scripts nix-update
+#!nix-shell -i bash -p coreutils gnugrep gnused common-updater-scripts nix-update
 
-version=$(list-git-tags | tail -1 | sed 's|^releases/||')
+version=$(list-git-tags | grep -E '^releases/[0-9]+\.[0-9]+\.[0-9]+$' | sed 's|^releases/||' | sort -rV | head -1)
```

Comment:
Also, I am not sure how strongly we can rely on the fact that all future versions will be of the form "X.Y.Z" with exactly three components. But it seems fine to not use a more convoluted regex until it actually turns out to be necessary.